docs/book/src/reference/versions.md

```diff
@@ -81,31 +82,33 @@ The Core Provider also talks to API server of every Workload Cluster. Therefore,
+|                                                                   | CAPI v1alpha3 (v0.3) Management | CAPI v1alpha3 (v0.3) Workload | CAPI v1alpha4 (v0.4) Management | CAPI v1alpha4 (v0.4) Workload | CAPI v1beta1 (v1.x) Management | CAPI v1beta1 (v1.x) Workload |
+|-------------------------------------------------------------------|---------------------------------|-------------------------------|---------------------------------|-------------------------------|--------------------------------|------------------------------|
+| Kubernetes v1.16 + kubeadm/v1beta2                                | ✓                               | ✓                             |                                 |                               |                                |                              |
+| Kubernetes v1.17 + kubeadm/v1beta2                                | ✓                               | ✓                             |                                 |                               |                                |                              |
+| Kubernetes v1.18 + kubeadm/v1beta2                                | ✓                               | ✓                             |                                 | ✓                             |                                | ✓                            |
+| Kubernetes v1.19 + kubeadm/v1beta2                                | ✓                               | ✓                             | ✓                               | ✓                             | ✓                              | ✓                            |
+| Kubernetes v1.20 + kubeadm/v1beta2                                | ✓                               | ✓                             | ✓                               | ✓                             | ✓                              | ✓                            |
+| Kubernetes v1.21 + kubeadm/v1beta2                                | ✓                               | ✓                             | ✓                               | ✓                             | ✓                              | ✓                            |
+| Kubernetes v1.22 + kubeadm/v1beta2 (v0.3) kubeadm/v1beta3 (v0.4+) |                                 | ✓                             | ✓                               | ✓                             | ✓                              | ✓                            |
+| Kubernetes v1.23 + kubeadm/v1beta3                                |                                 |                               | ✓                               | ✓                             | ✓                              | ✓                            |
+| Kubernetes v1.24 + kubeadm/v1beta3                                |                                 |                               | ✓                               | ✓                             | ✓                              | ✓                            |
```

Comment:
I assume we don't want to drop support for 1.24 in v0.4/v1alpha4 unless we have good reason to, right?